dist/media/package.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
Do we need to have the list of "files" in the package.json for them to be pushed to npm?